acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,1015 @@
+        LOGGER.info(actualText);
+    }
+
+    public void enterInValidIssueDate() {
+        new DrivingLicencePageObject().LicenceIssueDay.click();
+        new DrivingLicencePageObject().LicenceIssueDay.sendKeys("");
+        new DrivingLicencePageObject().LicenceIssueMonth.click();
+        new DrivingLicencePageObject().LicenceIssueMonth.sendKeys("");
+        new DrivingLicencePageObject().LicenceIssueYear.click();
+        new DrivingLicencePageObject().LicenceIssueYear.sendKeys("");
+    }
+
+    public void enterInValidIssueDateWithFutureYear() {
+        new DrivingLicencePageObject().LicenceIssueDay.clear();
+        new DrivingLicencePageObject().LicenceIssueDay.click();
+        new DrivingLicencePageObject().LicenceIssueDay.sendKeys("23");
+        new DrivingLicencePageObject().LicenceIssueMonth.clear();
+        new DrivingLicencePageObject().LicenceIssueMonth.click();
+        new DrivingLicencePageObject().LicenceIssueMonth.sendKeys("03");
+        new DrivingLicencePageObject().LicenceIssueYear.clear();
+        new DrivingLicencePageObject().LicenceIssueYear.click();
+        new DrivingLicencePageObject().LicenceIssueYear.sendKeys("2032");
+    }
+
+    public void issueDateErrorWelsh() {
+        String expectedText = "Gwall:\n" + "Rhaid i ddyddiad cyhoeddi fod yn y gorffennol";
+        String actualText = pastIssueDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void issueDateErrorDVAWelshh() {
+        String expectedText = "Gwall:\n" + "Rhaid i ddyddiad cyhoeddi fod yn y gorffennol";
+        String actualText = pastIssueDateErrorDVAMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void clickOnDVLARadioButtonWelsh() {
+        radioBtnDVLA.click();
+        CTButton.click();
+    }
+
+    public void clickOnDVARadioButtonWelsh() {
+        radioBtnDVA.click();
+        CTButton.click();
+    }
+
+    public void noDrivingLicenceOptionWelsh() {
+        noDLOption.click();
+        CTButton.click();
+    }
+
+    public void inValidDLText() {
+        invlidDLErrorText.isDisplayed();
+    }
+
+    public void inValidIssueDateText() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch y dyddiad fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = pastIssueDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void inValidIssueDateTextDVA() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch y dyddiad fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = expiryDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void enterInValidUntilDate() {
+        new DrivingLicencePageObject().LicenceValidToDay.click();
```

Comment:
Same here as above

Reply:
Changed to DrivingLicencePageObject drivingLicencePageObject = new DrivingLicencePageObject();